core/src/main/java/org/aerogear/mobile/core/MobileCore.java

```diff
@@ -37,18 +38,20 @@
 
     public static final String DEFAULT_CONFIG_FILE_NAME = "mobile-services.json";
 
-    private static final String TAG = "AEROGEAR/CORE";
     private static final int DEFAULT_READ_TIMEOUT = 30;
     private static final int DEFAULT_CONNECT_TIMEOUT = 10;
     private static final int DEFAULT_WRITE_TIMEOUT = 10;
+
+    @SuppressLint("StaticFieldLeak")
+    private static MobileCore instance;
```

Comment:
[Optional] Probably overlooked final here?

Reply:
I have no idea how to do that in a singleton static variable :/

My point is `init` is creating it, so... I think it can't be final!